packages/core/src/lib/markdown-it/index.js

```diff
@@ -14,7 +14,7 @@ const logger = require('../../utils/logger');
 
 const { HighlightRule } = require('./highlight/HighlightRule');
 
-const HIGHLIGHT_LINES_DELIMITER_REGEX = new RegExp(',(?![^\\[\\]]*])');
+const HIGHLIGHT_LINES_DELIMITER_REGEX = /,(?![^[\]]*])/;
```

Comment:
```suggestion
const HIGHLIGHT_LINES_DELIMITER_REGEX = /,(?![^\[\]]*])/;
```
@EltonGohJH this place might be missing a slash
![image](https://user-images.githubusercontent.com/41845017/178984613-b0916b70-2ecc-427d-b489-bd6adbca4ed8.png)
![image](https://user-images.githubusercontent.com/41845017/178984787-91b15b8a-7bc6-4db5-8557-fd6a88f4765e.png)

Reply:
@tlylt 
I get an eslint error when adding the extra escape character.
![image](https://user-images.githubusercontent.com/75515229/179399273-e83ffbce-3adc-4b0a-97ec-37ec35b4ebfe.png)

Reply:
Hi @EltonGohJH , looks like the original regex
`new RegExp(',(?![^\\[\\]]*])');` could have been
`new RegExp(',(?![^[\\]]*])');`

In that case, let's go with your updated one. Thanks for checking!